src/components/UserInfo/UserInfo.scss

```diff
@@ -0,0 +1,11 @@
+.UserInfo {
+  text-decoration: none;
+  text-transform: none;
+  font-size: 18px;
```

Comment:
Consider using `rem` or `em` instead of `px` for font size to make it more responsive. 📱